subprojects/gradle/runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/ArtifactsTestListener.kt

```diff
@@ -77,6 +77,7 @@ internal class ArtifactsTestListener(
                 remotePath = testMetadataDirectory.toPath()
             ).invoke()
         } catch (t: Throwable) {
+            // TODO LOST
```

Comment:
Can't report from here, maybe add special function to `TestLifecycleListener` and pass to `ReportViewerTestReporter`?

Reply:
finished it's a state so we should track the correct state
It could be done by returning a result of the state transition